### ISSUE TITLE
MoneroKon 2022 Planning Meeting: Sunday 6th February 2022 @ 18:00 UTC

### ISSUE THREAD

**Issue by umma08** [monero-project/meta#660]
**Location IRC:**
#monero-events

**The Matrix channel:**
#monero-events:monero.social (https://matrix.to/#/#monero-events:monero.social?via=matrix.org&via=monero.social)

**The Telegram channel:**
t.me/monerokon

**These instructions for joining the Matrix server:**
https://forum.monero.space/d/79-how-to-join-the-monero-core-team-matrix-server-web

**Time:**
 18:00 UTC

**Proposed Meeting Items :**
Please comment on GitHub in advance of the meeting if you would like to propose an agenda item.

**Agenda:**

1. Greetings
2. Update on CCS status
3. Update on availability/quotes (G+H)
4. Update on website/domain (ajs)
5. Updates from kanban board (if applicable)
6. Agree next steps
7. AOB
8. Confirm next meeting time and date

**Comment by escapethe3RA** [comment]
@umma08 I assume you wanted it to say *Sunday 6th* (4th is Friday), is that accurate?

I went with SUN the 6th in the Monero Observer Calendar: https://www.monero.observer/tag/calendar/

Ping me if something changes.

**Comment by umma08** [comment]
> @umma08 I assume you wanted it to say _Sunday 6th_ (4th is Friday), is that accurate?
> 
> I went with SUN the 6th in the Monero Observer Calendar: https://www.monero.observer/tag/calendar/
> 
> Ping me if something changes.

yeah - apologies it was Sunday the 6th. Brain freeze. Must have been looking at the wrong month in my calendar.

**Comment by umma08** [comment]
**Logs below**

,,,
[2022-02-06 18:00:03] <midipoet> And, it's time for today's MoneroKon planning meeting.
[2022-02-06 18:00:12] <midipoet> agenda can be found here: https://github.com/monero-project/meta/issues/660
[2022-02-06 18:00:15] <midipoet> 1. Greetings
[2022-02-06 18:00:33] <msvb-web> Hello
[2022-02-06 18:01:25] <deedledea[m]> hey there
[2022-02-06 18:01:32] <binaryFate> hello
[2022-02-06 18:02:00] <midipoet> 2. Update on CCS status
[2022-02-06 18:02:13] → kic0 joined (~kico@user/kico)
[2022-02-06 18:02:19] <ajs_[m]> hi
[2022-02-06 18:02:51] <midipoet> so, we have managed to raise 284 XMR of the total requested (328 XMR)
[2022-02-06 18:02:54] <midipoet> https://ccs.getmonero.org/proposals/MoneroKon-2022-CCS.html
[2022-02-06 18:03:15] <midipoet> which on the one hand is great - on the other, a little concerning
[2022-02-06 18:03:56] <midipoet> we set a deadline of mid-Feb for the full amount (otherwise we cancel and the funds rollover to 2023)
[2022-02-06 18:04:53] <kic0> hi
[2022-02-06 18:04:55] ⇐ kico quit (~kico@user/kico): Ping timeout: 256 seconds
[2022-02-06 18:04:58] <binaryFate> Maybe a reddit post to make a bit of noise?
[2022-02-06 18:05:19] <localmonero05> There have been 3 reddit posts in total, along with a tweet from @monero on Twitter.
[2022-02-06 18:05:32] <midipoet> yes, the last reddit post was the other day
[2022-02-06 18:05:38] <midipoet> https://www.reddit.com/r/Monero/comments/skoigt/ccs_funding_required_update_2_monerokon_2022/
[2022-02-06 18:05:55] <midipoet> for some reason i cannot see the comments - don't know if that is just me though
[2022-02-06 18:06:09] <localmonero05> https://nitter.it/monero/status/1486406276770840583
[2022-02-06 18:06:42] <binaryFate> I cannot see the comments either, except a single one from likely a bot
[2022-02-06 18:07:23] <midipoet> i know the post got auto-moderated for some reason, but i thought it was 'released'
[2022-02-06 18:07:33] <localmonero05> Loads of reddit posts get the comment count high, yet display just a few. Either a filter is hitting them or reddit itself is deleting. Unclear.
[2022-02-06 18:07:49] <localmonero05> Post got approved. It's live. Just not lots of engagement with the thread.
[2022-02-06 18:07:51] <midipoet> sethforprivacy: can you check this ^
[2022-02-06 18:07:56] <midipoet> please
[2022-02-06 18:07:58] <localmonero05> You can use teddit to double check. https://teddit.adminforge.de/r/Monero/comments/skoigt/ccs_funding_required_update_2_monerokon_2022/
[2022-02-06 18:09:11] <midipoet> i think there is a real chance that the CCS won't get funded, and more pressing, i really worry for the second CCS that we had planned (as appetite for the event seems low)
[2022-02-06 18:09:18] <midipoet> open to any and all ideas on this
[2022-02-06 18:10:12] <binaryFate> I'm not overly worried personally. Large price volatility often leads to everyone being in wait-and-see for a while
[2022-02-06 18:11:59] <ajs_[m]> down scaling the number of estimated attendees could be an option if it helps with reducing costs
[2022-02-06 18:12:26] * kic0 → kico
[2022-02-06 18:13:46] <msvb-web> One venue was just 50 capacity.
[2022-02-06 18:14:47] <midipoet> if we downscaled to 50 capacity, it would help some costs, surely - but fundamentally we are underdelivering on the what we said we would
[2022-02-06 18:15:06] <midipoet> it becomes a high-quality meetup, rather than a conference
[2022-02-06 18:17:01] <ajs_[m]> imo the caliber of talks is more important than the number of attendees
[2022-02-06 18:17:59] <deedledea[m]> especially if streamed to virtual attendees
[2022-02-06 18:19:36] <sgp_[m]1> Ticket sales are always another form of revenue
[2022-02-06 18:19:56] <midipoet> ok. so i guess we can table this for next week (the 13th). shall we set the deadline for the CCS being funded to that date?
[2022-02-06 18:20:29] <midipoet> sgp_[m]1: the downscaled event wouldn't leave much room for ticket sales. 
[2022-02-06 18:21:01] <sgp_[m]1> Sure, 50 seems very small tbh
[2022-02-06 18:22:13] <binaryFate> to be pedantic if we said mid-feb, it's weird to put deadline earlier
[2022-02-06 18:22:39] <localmonero05> Just 2 days earlier? Or what would be mid-feb for you? 15th?
[2022-02-06 18:22:56] <midipoet> 14th and half (leap year)
[2022-02-06 18:23:20] <localmonero05> Is it leap year? Nice. Anyways. I guess midipoet was shooting for 13th since it's a meeting day.
[2022-02-06 18:24:04] <midipoet> localmonero05: yes. i figured we could try and make a decision then, based on the most up to date/final knowledge of the CCS
[2022-02-06 18:24:38] <localmonero05> I am in favor of that.
[2022-02-06 18:26:02] <msvb-web> Do you have another date in mind binaryFate?
[2022-02-06 18:27:02] <binaryFate> I can imagine a scenario where we irrevocably decide to cancel on the 13th, and some donors feel we did not follow up promise and should have left couple more days to mid-feb.
[2022-02-06 18:27:40] <binaryFate> but that's unlikely and overly pedantic, I'm fine with deciding on the 13th
[2022-02-06 18:27:58] <binaryFate> if amount is near completion at that time, we may want to give it few more days 
[2022-02-06 18:28:31] <midipoet> yes - let's see where we are next Sunday (13th), but i do think we may have to brace ourselves for a) cancelling b) downscaling
[2022-02-06 18:28:41] <ajs_[m]> agree
[2022-02-06 18:28:52] <midipoet> 2. Update on availability/quotes (G+H)
[2022-02-06 18:29:03] <binaryFate> sorry one quick question
[2022-02-06 18:29:17] <midipoet> binaryFate: go for it
[2022-02-06 18:29:33] <binaryFate> are there other factors besides the CCS funding amount that lead some of you to think of canceling/downscaling?
[2022-02-06 18:32:21] <msvb-web> No other factors for the early cancellation.
[2022-02-06 18:32:39] <midipoet> i think for me, the general hesitancy/lack of community appetite for the event has made me question whether the second CCS would be successful - which made me worry we might be left with everything in place/organised - but with a serious whole in the XMR available to get speakers/volunteers over.
[2022-02-06 18:33:00] <msvb-web> Obviously public health could cause cancelation, bit later on.
[2022-02-06 18:33:50] <midipoet> i was also worried that the price volatility would seriously damage our intended goals (it has to some extent already)
[2022-02-06 18:35:11] <binaryFate> I do perceive a lack of community appetite all around Monero (at least compared to engagement quality as few years ago) in general, not just about the konferenco. Hence why we need real life events more than ever IMO.
[2022-02-06 18:36:47] <midipoet> ok. maybe we should try for some more tweets/reddit posts. can someone else do one, as i am sales pitched out of it?
[2022-02-06 18:40:08] <binaryFate> I'm not much on twitter but I can tweet something
[2022-02-06 18:41:44] <midipoet> ok - let's move onto
[2022-02-06 18:41:45] <midipoet> 3. Update on availability/quotes (G+H)
[2022-02-06 18:42:03] <midipoet> i don't think @manhuichi or @gramsdb are here to update. 
[2022-02-06 18:42:32] <midipoet> they postponed the mid-week meeting last week, as they were still waiting on formal quotes from both Suspenso and Arroz
[2022-02-06 18:42:44] <midipoet> we are aiming for a mid-week meeting this week
[2022-02-06 18:42:56] <midipoet> probably Wednesday
[2022-02-06 18:43:12] <msvb-web> Suspenso is the classy one.
[2022-02-06 18:43:17] <midipoet> in the meantime, i think it would be wise to ask them to more formally price the 50 person venue
[2022-02-06 18:44:02] <ajs_[m]> in or out doors?
[2022-02-06 18:44:47] <midipoet> ajs_[m]: i would imagine indoors, with an option of an outside space (maybe shared)
[2022-02-06 18:45:29] <ajs_[m]> okay
[2022-02-06 18:46:29] <midipoet> is everybody ok for Wednesday this week for a meeting with them (i suggest 18 UTC)
[2022-02-06 18:47:08] <deedledea[m]> works for me
[2022-02-06 18:47:09] <binaryFate> ok
[2022-02-06 18:48:04] <midipoet> i will ping everyone once i have a confirmation from @manhuichi and @gramsdb
[2022-02-06 18:48:20] <midipoet> 4. Update on website/domain (ajs)
[2022-02-06 18:48:33] <ajs_[m]> I am still waiting on brandon to update the DNS to point to the new site
[2022-02-06 18:49:06] <ajs_[m]> he mentioned that he was dealing with stuff. I informed him that the website is low priority right, so he could get to it whenever it's convenient
[2022-02-06 18:49:08] <msvb-web> Okay for me on Wednesday for the meeting.
[2022-02-06 18:49:38] <midipoet> ajs_[m]: ok. is there any other issue on the website we need to discuss?
[2022-02-06 18:49:45] <midipoet> msvb-web: thanks for confirming
[2022-02-06 18:50:01] <ajs_[m]> that is all for website
[2022-02-06 18:50:12] <midipoet> ok
[2022-02-06 18:50:19] <midipoet> 5. Updates from kanban board (if applicable)
[2022-02-06 18:50:34] <midipoet> i don't think there are any updates to this.
[2022-02-06 18:50:42] <midipoet> though i think the matrix bridge is still down
[2022-02-06 18:50:48] <midipoet> (does anybody know)
[2022-02-06 18:50:54] <midipoet> so many carrington[m] has an update
[2022-02-06 18:50:59] <midipoet> but can't be pinged
[2022-02-06 18:51:11] <msvb-web> Ajs is a maze,
[2022-02-06 18:51:30] <msvb-web> Matrix user, I wanted to say.
[2022-02-06 18:51:38] <midipoet> msvb-web: ah yes
[2022-02-06 18:51:46] <ajs_[m]> yes, i am using matrix right now
[2022-02-06 18:51:48] <ajs_[m]> no issues
[2022-02-06 18:51:56] <msvb-web> So maybe it's working?
[2022-02-06 18:52:24] <midipoet> ok - so i guess we can move to 6. Agree next steps
[2022-02-06 18:52:35] <ajs_[m]> msvb-web: i thought you meant amazing, lol
[2022-02-06 18:53:04] <midipoet> which is essentially the midweek meeting and plan/brace for potential downscale/cancel
[2022-02-06 18:53:28] <midipoet> is there anything else that we should complete before next week?
[2022-02-06 18:56:17] <midipoet> 7. AOB (any other business)?
[2022-02-06 18:59:50] <midipoet> ok. i think we can call the meeting
[2022-02-06 19:00:10] <ajs_[m]> thanks midipoet 
[2022-02-06 19:00:14] <binaryFate> thank you
[2022-02-06 19:01:10] <deedledea[m]> thanks! catch some of you on Wednesday
[2022-02-06 19:01:23] <midipoet> thanks all. see you next week (either Wednesday or Sunday or both)
[2022-02-06 19:02:02] <msvb-web> Great meeting, thanks for the good moderation.
,,,
